src/modules/show-comments.js

```diff
@@ -0,0 +1,21 @@
+function showCommentsList(id, parent) {
+  const apiKey = 'tnE2k6P5BdZ2HCTjbd0V';
+  fetch(
+    `https://us-central1-involvement-api.cloudfunctions.net/capstoneApi/apps/${apiKey}/comments?item_id=${id}`,
+  )
+    .then((response) => response.json())
+    .then((data) => {
+      data.forEach((comment) => {
+        const title = document.createElement('h3');
+        title.className = 'meal-popup-comments-title';
```

Comment:
- [ ] Sometimes when I open the comments popup, the following runtime error may show up.  kindly consider to use do this work using async and await and also try to use async and await in other similar instances in your app. the reason for this error is that the forRach method ry to run on data before even it gets fetched so running forEach on null results in an error.